weave

```diff
@@ -1945,10 +1949,13 @@ EOF
     attach-router)
         check_running $CONTAINER_NAME
         enforce_docker_bridge_addr_assign_type
+        # We cannot use detect_awsvpc here, because HTTP server might not be started
```

Comment:
I have it in mind, but will do it in a separate PR once you are happy with the check.